### ISSUE TITLE
Incorrect tally of A in player's hand

### ISSUE THREAD

**Issue by adamdiy** [duckRabbitPy/vanillaJS-blackjack#19]
Should be 11 in this case: looks like it is counted as 1    :)

![image](https://user-images.githubusercontent.com/8769879/143440182-eba4dbc6-91e0-46e8-ad25-215143bc113f.png)

**Comment by duckRabbitPy** [comment]
Hi @adamdiy, 

Thanks for checking out the game! I'm not sure if this is an 'official rule' but when I have played with friends we have counted Aces as low if you were due to go bust with high aces, so this is intended behaviour outlined in rule 4. There is a seperate function sumLowAces to deal with this situation and makes it possible to get 5 cards under 21.

In help section: 
4. Picture cards are worth 10. Aces count as 11 (unless you are going to go bust with 11 in which case they will be worth 1)
